Sources/SwiftDriver/Explicit Module Builds/ExplicitModuleBuildHandler.swift

```diff
@@ -29,11 +29,25 @@ public struct ExplicitModuleBuildHandler {
   private var swiftModuleBuildCache: [ModuleDependencyId: Job] = [:]
 
   /// The toolchain to be used for frontend job generation.
-  private let toolchain: Toolchain
+  private let toolChain: Toolchain
```

Comment:
The type is Toolchain, I think they should be consistent.
```suggestion
  private let toolchain: Toolchain
```